recipe/meta.yaml

```diff
@@ -30,11 +30,10 @@ requirements:
     - jedi >=0.10
     - pexpect  # [unix]
     - pickleshare
-    - prompt_toolkit >=2.0.0,<3
+    - prompt_toolkit >=2.0.0,<4
     - pygments
     - python
     - traitlets >=4.2
-    - win_unicode_console >=0.5  # [win and py<36]
```

Comment:
did we somehow internalize this feature?

Reply:
oh, no seeing the bit about not supporting 3.5...

Reply:
No we dropped 3.5

Reply:
add a skip up in build!